data/policies/parent_call.cas

```diff
@@ -0,0 +1,36 @@
+virtual resource bar {
+	fn read(domain source) {
+		allow(source, this, file, read);
+	}
+}
+
+resource foo inherits bar {
+	fn read(domain source) {
+		allow(source, this, dir, read);
+	}
+}
+
+domain dom {
+	foo<bar>.read(this);
+}
+
+virtual resource abc {
+	fn read(domain source) {
+		allow(source, this, file, read);
+	}
+}
+
+resource xyz inherits abc {
+	fn read(domain source) {
+		abc.read(source);
+	}
+}
+
+// TODOs
```

Comment:
Are there TODOs in the code for what's left to do functionality wise to support these use cases?

Reply:
The TODO added in one of your previous comments should cover both of these.

Reply:
But this does bring up another question.  Do we want <bar>.read(this); to work with this PR?  Currently this is broken at the parser level.

Reply:
What do you mean by ".read(this)"?  Not sure I follow the question.

Reply:
Sorry markup ate the comment see edit above

Reply:
That makes more sense, but I'm still not sure I follow.  What behavior is `<bar>.read(this);` supposed to enable?  In the test case it looks like it should just be `xyz.read(this);` without brackets?  What's the difference between `<bar>.read(this);` and `bar.read(this)`?